### PR TITLE
[prometheus] Add naseemkullah as maintainer

### PR DESCRIPTION
**Pull request by naseemkullah**
👋 
I've made a few contributions overtime and we have been using this chart extensively in production at Transit for over 2 years.

I'd like to look into eventually simplifying/cleaning up the chart and modularizing it by using node-exporter, pushgateway and potentially alertmanager subcharts.

### CODE REVIEW

File: charts/prometheus/Chart.yaml [hunk 1]
```diff
@@ -20,5 +20,7 @@ maintainers:
     email: miroslav.hadzhiev@gmail.com
   - name: monotek
     email: monotek23@gmail.com
+  - name: naseem
```
Comment (scottrigby) [inline charts/prometheus/Chart.yaml:23]: Also add to /.github.com/CODEOWNERS

**Review by scottrigby — CHANGES_REQUESTED** [review]

**Comment by scottrigby** [comment]
I support this. Naseem helped get this repo migration conversation started with prometheus devs earlier in the year, and would be very helpful collaborating with you all and prom devs to further simplify the chart.

**Review by monotek — DISMISSED** [review]
Thumps up from me too 😎

**Comment by Xtigyro** [comment]
@naseemkullah I would appreciate if you could msg me on the Kubernetes Community Slack workspace - Miro Hadzhiev - so we can have a small chat and get to know each other before proceeding with this PR.

**Review by Xtigyro — APPROVED** [review]

**Comment by Xtigyro** [comment]
@monotek There were some changes pushed - I would like to ask for a re-approval.

**Review by monotek — APPROVED** [review]

**Comment by Xtigyro** [comment]
@scottrigby Your finishing shot please, he-he-he 🥇

**Review by zanhsieh — APPROVED** [review]

**Review by scottrigby — APPROVED** [review]

**Pull request merged**